<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwf2main</class>
 <widget class="QMainWindow" name="pwf2main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>101</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>221</width>
    <height>101</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>221</width>
    <height>101</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pwf2tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pwf2main.qrc">
    <normaloff>:/img/pwf2tools.png</normaloff>:/img/pwf2tools.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pwf2int">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>pwf2int</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pwf2spm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>pwf2spm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pwf2tex">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>pwf2tex</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionConsole"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionpwf2ext">
   <property name="text">
    <string>pwf2ext</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pwf2main.qrc"/>
 </resources>
 <connections/>
</ui>
